### PR TITLE
Add com.jagex.Launcher

### PR DESCRIPTION
**Pull request by nmlynch94**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

### CODE REVIEW

File: com.jagex.Launcher.yml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      - type: file
+        path: icons/32/32.png
+      - type: file
+        path: icons/48/48.png
+      - type: file
+        path: icons/64/64.png
+      - type: file
+        path: icons/128/128.png
+      - type: file
+        path: icons/256/256.png
```
Comment (hfiguiere) [inline com.jagex.Launcher.yml:150]:
```suggestion
```

remove all of these and keep the 512x512

Comment (hfiguiere) [reply com.jagex.Launcher.yml:150]: including the files

**Review by hfiguiere — COMMENTED** [review]
there is something against packaging Windows applications on flathub.

**Comment by nmlynch94** [comment]
As in it's not allowed?

**Comment by nmlynch94** [comment]
Ok, got it, thanks. I see a couple apps like https://github.com/goatcorp/FFXIVQuickLauncher that are launchers for wine apps that they did not author. Are launchers OK in principle as long as the launcher itself is native before launching the wine application?

**Pull request closed without merging**